CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ Notable changes.
 
 ## September 2022
 
+### [0.18.0]
```

Comment:
Should this be added under `October 2022`?

Reply:
Yes, let's have it under October.

Reply:
Adde 👍